apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/implementation/redirect.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Redirect'
+description: 'Recipe for handling redirects using xmcloud and vercel.'
+hasSubPageNav: true
+hasInPageNav: false
+area: ['accelerate']
+---
+
+## Problem
+
+The customer has the following requirements:
+
+- R1. A marketer, strategist, or content author would like to create redirects.
+- R2. The Customer has a huge list of redirects after migration to xmcloud.
```

Comment:
Change 'Huge' to 'large' and remove the limitation of "after migration to XM Cloud"